db/schema.rb

```diff
@@ -570,6 +594,138 @@
      FROM (appointments ap
        JOIN facilities f ON ((ap.facility_id = f.id)));
   SQL
+  create_view "latest_blood_pressures_per_patient_per_days", materialized: true, sql_definition: <<-SQL
+      WITH latest_bp_per_patient_per_month AS (
```

Comment:
I regenerated the schema locally on this branch to see if there were any diffs. The only issue I found is that this view is wrong. I reckon you ran the migration that created this view while it was still in development. During its development we found a bug in the view and [fixed it here](https://github.com/simpledotorg/simple-server/commit/a2eb5bd14abab46a39cea33d10c3f4a87aef36fa#diff-46826eac74e5c6efde6ce2c953d2ca03). 

I will push an update.

The other changes in here look legitimate. Looks like our schema was out of date?